Products/Zuul/infos/metricserver.py

```diff
@@ -136,6 +136,20 @@ def __init__(self, graph, context):
         self._contextRPN = False
         super(ColorMetricServiceGraphPoint, self).__init__(graph, context)
 
+    @property
+    def id(self):
+        if self._multiContext:
+            dev = self._context.device()
+            if dev and dev.id != self._context.id:
+                return "%s %s %s" % (dev.id, self._context.id, self._object.id)
+            else:
+                return "%s %s" % (self._context.id, self._object.id)
+        return self._object.id
+
+    @property
+    def name(self):
```

Comment:
I'm concerned that you changed the value for the name property.  I believe this value gets used to calculate RPN expressions.